src/unstract/sdk/file_storage/helper.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Any
+
+import fsspec
+from google.oauth2 import service_account
+
+from unstract.sdk.exceptions import FileStorageError
+from unstract.sdk.file_storage.constants import GCS
+from unstract.sdk.file_storage.fs_provider import FileStorageProvider
+
+logger = logging.getLogger(__name__)
+
+
+class FileStorageHelper:
+    @staticmethod
+    def file_storage_init(provider: FileStorageProvider, credentials: dict[str, Any]):
+        """Initialises file storage based on provider.
+
+        Args:
+            provider (FileStorageProvider): Provider
+            credentials (dict): Credentials based on provider
+
+        Returns:
+            NA
+        """
+        if provider == FileStorageProvider.GCS:
+            return FileStorageHelper.gcs_init(credentials)
+        elif provider == FileStorageProvider.Local:
+            return FileStorageHelper.local_file_system_init()
+
+    @staticmethod
+    def gcs_init(credentials):
```

Comment:
@gaya3-zipstack Looks like the pre-commit is not run over this commit. Please add the return type to method